docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
 
-- Click on **+ Add New Categories** button. Type `Brunch` in the **Category** field. **DO NOT ADD** `Biscotte Restaurant` to the `Restaurants` dropdown on the right.
-- Click **Save**.
+In the right sidebar, under **Categories (1)**, click on **Add an item…** and select `Brunch`.  Click **Save**.
 
-You will see the **Brunch** category listed in the entries.
+### Step 3: Set Roles & Permissions
 
-Navigate back to [**COLLECTION TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant).
+1. Click on _General > Settings_ at the bottom of the left-hand menu.
+2. Under _Users & Permissions Plugin_, choose [**Roles**](http://localhost:1337/admin/settings/users-permissions/roles).
+3. Click the **Public** role.
+4. Scroll down under **Permissions**.
+5. In the **Application** tab, find **Restaurant**. 
+6. Click the checkboxes next to **find** and **findone**.
+7. Repeat with **Category**: click the checkboxes next to **find** and **findone**.
+8. Finally, click **Save**.
 
-- Click on `Biscotte Restaurant`.
-- On the right, under **Categories(1)**, `select` the `Add an item...`, and add **Brunch** as a category for this restaurant, and click the **Save** button.
+![Screenshot: Public Role in Users & Permissions plugin](../assets/quick-start-guide/qsg-handson-part2-04-roles.png)
 
-You have now seen **two different ways** to use the **relation** field type to add and connect relations between Content Types.
+### Step 4: Publish the content
 
-## 7. Set Roles and Permissions
+By default, any content you create is saved as a draft. Let's publish our entries in the collection types.
 
-Navigate to [**GENERAL** > **Settings** > **USERS & PERMISSIONS PLUGIN** > **Roles**](http://localhost:1337/admin/settings/users-permissions/roles).
+First, navigate to [_Collection types > Categories_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category). From there:
 
-- Click the **Public** Role.
-- Scroll down under **Permissions**, open the **Application** tab and find **Restaurant**. Click the checkbox next to **find** and **findone**.
-- Repeat and find **Category**. Click the checkbox next to **find** and **findone**.
-- Click **Save**.
+1. Click the **Draft** button on the `Brunch` category.
+2. On the next screen, click **Publish**.
+3. In the **Please confirm** dialog, click **Yes, publish**.  
 
-## 8. Publish the content
+Then, go back to the Categories list and repeat for the `French food` category.
 
-By default, any content you create is saved as a draft. To publish your content:
+Finally, to publish your `Biscotte Restaurant`, go to [_Collection types > Restaurants_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant), click the **Draft** button for the entry, and **Publish** the restaurant entry.
```

Comment:
Same here about the Draft button.
Still strange for me.

Reply:
See reply to [comment above](https://github.com/strapi/documentation/pull/288#discussion_r641927957). I can update it if needed :-)